package.json

```diff
@@ -93,6 +93,7 @@
     "jquery": "1.12.3",
     "json-loader": "0.5.4",
     "json-stable-stringify": "1.0.1",
+    "jsx-to-string": "1.3.1",
```

Comment:
Could you run `npm run update-deps` to update also `npm-shrinkwrap.json` file?

https://github.com/Automattic/wp-calypso/blob/5e1fb8eeeb8d038e0a2b15202c23b3edb3fe75fd/docs/shrinkwrap.md

Reply:
Done!

Reply:
whoops, done